catalog/istio-stack/kiali/releases.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: kiali-operator
+  namespace: istio-system
+spec:
+  interval: 10m0s
+  chart:
+    spec:
+      chart: kiali-operator
+      version: "1.63.1"
+      sourceRef:
+        kind: HelmRepository
+        name: kiali
+      interval: 1m
+  install:
+    crds: CreateReplace
+  upgrade:
+    crds: CreateReplace
+  values:
+    onlyViewOnlyMode: true
+    watchNamespace: ""
+
+    # For what a Kiali CR spec can look like, see:
+    # https://github.com/kiali/kiali-operator/blob/master/crd-docs/cr/kiali.io_v1alpha1_kiali.yaml
+    cr:
+      create: true
+      name: kiali-operator
+      namespace: "istio-system"
+      spec:
+        installation_tag: "Kiali - View Only"
+        istio_namespace: "istio-system"
+        auth:
+          strategy: "anonymous"
+        deployment:
+          accessible_namespaces:
+            - '**'
+          view_only_mode: true
+          instance_name: "kiali"
+        external_services:
+          custom_dashboards:
+            enabled: true
+            is_core: false
+          grafana:
```

Comment:
but probably extract the two urls in postBuild.subsitution to make it easy to override?
can one kustomization override the postBuild.subsitutions of another?

Reply:
good question, but you could also just patch the helmrelease via kustomize

Reply:
> can one kustomization override the postBuild.subsitutions of another?

not sure, but probably not.
maybe we can take out these specific urls and let the user override them according to their setup?

Reply:
sure ;) postBuild.substitutions just would be simpler - but I also guess it won't work.

Reply:
if i want to overwrite something, I patch the HelmRelease